packages/clients/meldemichel/src/plugins/Gfi/Feature.vue

```diff
@@ -1,17 +1,11 @@
 <template>
   <v-card>
-    <v-icon
-      v-if="hasWindowSize && hasSmallWidth"
-      class="meldemichel-gfi-grip-icon"
-    >
-      fa-grip-lines
-    </v-icon>
-    <v-card-actions>
+    <v-card-actions v-if="!hasWindowSize || !hasSmallWidth">
       <!-- TODO implement when vector clusters are done
       <v-btn>Prev</v-btn>
       <v-btn>Next</v-btn>
       -->
-      <v-spacer></v-spacer>
+      <v-spacer />
```

Comment:
Both ways to model this are currently in the code base. If it matters to you, there's probably a linting rule that would autofix this globally for us. You e.g. added it in `packages/components/MoveHandle.vue` in HTML style, but seem to prefer self-closing?

🎩 since it doesn't really matter.